ext/app/electron/main.ts

```diff
@@ -72,15 +53,10 @@ class GristApp {
   private onStartup: any = null;
   private credential: string = makeId();
   private shouldQuit = false;
-  private authMode: GristElectronAuthMode;
+  private authMode: GristDesktopAuthMode;
 
   public constructor() {
-    const mode = process.env.GRIST_ELECTRON_AUTH;
-    if (mode === 'strict' || mode === 'none' || mode === 'mixed') {
-      this.authMode = mode;
-    } else {
-      this.authMode = 'strict';
-    }
+    this.authMode = process.env.GRIST_DESKTOP_AUTH as GristDesktopAuthMode;
```

Comment:
Could you do me a favor and either support `GRIST_ELECTRON_AUTH` as an no-longer-documented synonym for the new `GRIST_DESKTOP_AUTH` (my preferred option), or detect its presence and throw an error message explaining the change (acceptable option).  In general the threshold for breaking user setups should be high. Otherwise, given all the changes made to Grist every week, using Grist would become a serious chore.

Reply:
Honestly I wouldn't rename this if not for consistency. I'll do the former for now, since the big plan is to phase out this use pattern. As we discussed, it's probably a good idea to write a separate "server mode" for those who mainly use Desktop to launch a local Grist server on Windows. `GRIST_DESKTOP_AUTH` wouldn't make sense should that be done. Does that sound right?

Reply:
The former sounds fine: continuing supporting `GRIST_ELECTRON_AUTH` as a synonym for `GRIST_DESKTOP_AUTH`. Agree the feature may evaporate in future.